### DISCUSSION TITLE
Suggest a primary nav variation with an embedded search

### DISCUSSION THREAD

**Discussion by wesleymiles** [RedHat-UX/red-hat-design-system#1624] [Ideas]
docs.redhat.com will be launching an update soon with search exposed in the navigation by default (except on the homepage.) 
![image](https://github.com/RedHat-UX/red-hat-design-system/assets/613815/68949200-5c8e-400a-86a0-2bce09f38987)

With catalog and console having similar implementations, it might be worth exploring a design system solution.  
![image](https://github.com/RedHat-UX/red-hat-design-system/assets/613815/5daad7a2-5ebe-4448-97bc-46f076f5713c)

![image](https://github.com/RedHat-UX/red-hat-design-system/assets/613815/6054de71-0f9f-4239-ac5e-48415b5ca5be)

**Comment by bennypowers** [comment]
seems like a pattern which nav should support, but not necessarily a variant

@wesleymiles sorry if that sounds picayune. "pattern" and "variant" have very specific meanings in RHDS

**Reply by bennypowers** [reply]
https://github.com/orgs/RedHat-UX/discussions/1624
https://github.com/RedHat-UX/red-hat-design-system/issues/868
https://github.com/RedHat-UX/red-hat-design-system/issues/375
https://github.com/RedHat-UX/red-hat-design-system/issues/310

**Comment by bennypowers** [comment]
Since this is slated for work, i'll close this discussion here
